src/localization/fr-FR.json

```diff
@@ -0,0 +1,150 @@
+  "todays-shipping-cost": "T-o-d-a-y-&-a-p-o-s-;-s- -S-h-i-p-p-i-n-g- -C-o-s-t",
+  "todays-taxes": "T-o-d-a-y-&-a-p-o-s-;-s- -T-a-x-e-s",
+  "todays-total": "T-o-d-a-y-&-a-p-o-s-;-s- -T-o-t-a-l",
+  "total-quantity": "T-o-t-a-l- -Q-u-a-n-t-i-t-y",
+  "todays-subtotal": "T-o-d-a-y-&-a-p-o-s-;-s- -S-u-b-t-o-t-a-l",
+  "purchase-history": "P-u-r-c-h-a-s-e- -H-i-s-t-o-r-y",
+  "purchase-number": "P-u-r-c-h-a-s-e- -#",
+  "date": "D-a-t-e",
+  "total": "T-o-t-a-l",
+  "status": "S-t-a-t-u-s",
+  "no-previous-purchase-message": "Y-o-u- -h-a-v-e- -n-o- -p-r-e-v-i-o-u-s- -p-u-r-c-h-a-s-e-s-.",
+  "cancelled": "C-a-n-c-e-l-l-e-d",
+  "completed": "C-o-m-p-l-e-t-e-d",
+  "in-progress": "I-n- -P-r-o-g-r-e-s-s",
+  "new-payment-method": "N-e-w- -P-a-y-m-e-n-t- -M-e-t-h-o-d",
+  "failed-to-save-message": "F-a-i-l-e-d- -t-o- -S-a-v-e-,- -p-l-e-a-s-e- -c-h-e-c-k- -a-l-l- -r-e-q-u-i-r-e-d- -f-i-e-l-d-s- -a-r-e- -f-i-l-l-e-d-.",
+  "card-type": "C-a-r-d- -T-y-p-e",
+  "american-express": "A-m-e-r-i-c-a-n- -E-x-p-r-e-s-s",
+  "mastercard": "M-a-s-t-e-r-c-a-r-d",
+  "visa": "V-i-s-a",
+  "card-holders-name": "C-a-r-d- -H-o-l-d-e-r-&-a-p-o-s-;-s- -N-a-m-e",
+  "credit-card-number": "C-r-e-d-i-t- -C-a-r-d- -N-u-m-b-e-r",
+  "expiry-date": "E-x-p-i-r-y- -D-a-t-e",
+  "security-code": "S-e-c-u-r-i-t-y- -C-o-d-e",
+  "save-payment-to-profile": "S-a-v-e- -t-h-i-s- -p-a-y-m-e-n-t- -m-e-t-h-o-d- -t-o- -m-y- -p-r-o-f-i-l-e",
+  "continue": "C-o-n-t-i-n-u-e",
+  "none-available": "N-o-n-e- -A-v-a-i-l-a-b-l-e",
+  "original-price": "O-r-i-g-i-n-a-l- -P-r-i-c-e",
+  "price": "P-r-i-c-e",
+  "add-to-cart": "A-d-d- -t-o- -C-a-r-t",
+  "results": "r-e-s-u-l-t-s",
+  "results-on-page": "r-e-s-u-l-t-s- -o-n- -p-a-g-e",
+  "previous": "P-r-e-v-i-o-u-s",
+  "page": "p-a-g-e",
+  "of": "o-f",
+  "next": "N-e-x-t",
+  "product-recommendations": "P-r-o-d-u-c-t- -R-e-c-o-m-m-e-n-d-a-t-i-o-n-s",
+  "no-saved-address-message": "Y-o-u- -h-a-v-e- -n-o- -s-a-v-e-d- -a-d-d-r-e-s-s-e-s",
+  "addresses": "A-d-d-r-e-s-s-e-s",
+  "personal-information": "P-e-r-s-o-n-a-l- -I-n-f-o-r-m-a-t-i-o-n",
+  "no-payment-methods-message": "Y-o-u- -h-a-v-e- -n-o- -s-a-v-e-d- -p-a-y-m-e-n-t-M-e-t-h-o-d-s-.",
+  "payment-methods": "P-a-y-m-e-n-t- -M-e-t-h-o-d-s",
+  "sub-total": "S-u-b---t-o-t-a-l",
+  "tax": "T-a-x",
+  "item-total": "I-t-e-m- -T-o-t-a-l",
+  "summary": "S-u-m-m-a-r-y",
+  "order-number": "O-r-d-e-r- -N-u-m-b-e-r-:",
+  "order-tax-total": "O-r-d-e-r- -T-a-x- -T-o-t-a-l",
+  "order-purchase-date": "O-r-d-e-r- -P-u-r-c-h-a-s-e- -D-a-t-e",
+  "order-total": "O-r-d-e-r- -T-o-t-a-l",
+  "items": "I-t-e-m-s",
+  "register-new-account": "R-e-g-i-s-t-e-r- -a- -N-e-w- -A-c-c-o-u-n-t",
+  "email-slash-username": "E-m-a-i-l-/-U-s-e-r-n-a-m-e",
+  "password-confirmation": "P-a-s-s-w-o-r-d- -C-o-n-f-i-r-m-a-t-i-o-n",
+  "search-results": "S-e-a-r-c-h- -R-e-s-u-l-t-s",
+  "no-results-found": "N-o- -r-e-s-u-l-t-s- -f-o-u-n-d",
+  "expexted-release-date": "E-x-p-e-c-t-e-d- -R-e-l-e-a-s-e- -D-a-t-e"
+}
```

Comment:
uh oh, no newline at the end of file 😮 I recommend using this setting in VSCode: "files.insertFinalNewline": true,

Reply:
This file is auto generated, but I've added `\n` in the script that generates it.